airflow/models.py

```diff
@@ -1801,6 +1801,50 @@ def __init__(self, event, task_instance, owner=None, extra=None, **kwargs):
         self.owner = owner or task_owner
 
 
+class SkipMixin(object):
+    def skip(self, dag_run, execution_date, tasks):
```

Comment:
Does it make more sense to take in task_ids to be more flexible since we just convert it from the get-go? Not super opinionated here but might make some other cases simpler.